app/routes/settings/email-preferences.js

```diff
@@ -8,5 +8,10 @@ export default Route.extend({
   },
   model() {
     return this.get('authManager.currentUser').query('emailNotifications', { include: 'event' });
+  },
+  actions: {
+    willTransition() {
+      this.get('authManager.currentUser').rollbackAttributes();
```

Comment:
Not the user. We're not doing any changes to the user. All the email notifications records have to be rolled back.